requirements/dev.txt

```diff
@@ -35,7 +35,7 @@ fs==2.4.11                # via xblock
 future==0.18.2            # via pyjwkest
 idna==2.9                 # via requests
 importlib-metadata==1.6.1  # via importlib-resources, inflect, path, pluggy, pytest, tox, virtualenv
-importlib-resources==1.5.0  # via virtualenv
```

Comment:
I am concerned about this upgrade.

Reply:
Apparently, because https://github.com/edx/completion/blob/master/setup.py#L89, this is not a concern because when edx-completion got installed into edx-platform, we will still use `importlib-resources` version 1.5.0.